src/content-script.ts

```diff
@@ -4,14 +4,47 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+const regexPathName = /\/.+\/.+\/(issues|pull)\/\d+/;
+const replySummaryId = "saved-reply-new_comment_field";
+const footprint = "refined-saved-replies";
```

Comment:
[Naming] Just for consistency, maybe `footprintAttribute` because the previous variable is `replySummaryId`?

```suggestion
const footprintAttribute = "refined-saved-replies";
```

Reply:
Ooh also I just thought of this - maybe instead of a DOM attribute, there can be a variable set to the last URL that was detected & modified for? That way we can avoid the DOM...